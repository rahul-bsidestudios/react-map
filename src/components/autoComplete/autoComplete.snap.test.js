import React from 'react';
import renderer from 'react-test-renderer';
import MapboxAutoComplete from './index';

const mockSelect = jest.fn();
const mockClear = jest.fn();
const mockChange = jest.fn();

test('renders AutoComplete component', () => {
  const tree = renderer.create(<MapboxAutoComplete onSuggestionSelect={mockSelect} onChange={mockChange} clear={mockClear} />).toJSON();
  expect(tree).toMatchSnapshot();
});