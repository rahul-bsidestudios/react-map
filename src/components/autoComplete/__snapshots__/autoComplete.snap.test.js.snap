// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AutoComplete component 1`] = `
<div>
  <input
    autoComplete="off"
    className="react-mapbox-ac-input"
    onChange={[Function]}
    onKeyDown={[Function]}
    placeholder="Search"
    type="text"
  />
  <span>
    <div
      className="react-mapbox-ac-menu"
      onClick={[Function]}
      style={
        Object {
          "display": "none",
        }
      }
    >
      
    </div>
  </span>
</div>
`;
